vllm/v1/core/kv_cache_manager.py

```diff
@@ -22,7 +22,7 @@ class KVCacheBlocks:
     Scheduler and KVCacheManager, to hide KVCacheManager's internal data
     structure from the Scheduler.
     """
-    blocks: tuple[list[KVCacheBlock], ...]
+    blocks: tuple[tuple[KVCacheBlock, ...], ...]
```

Comment:
Can you add a comment for 
* `tuple[tuple[]]` -> avoid creating empty list
* `get_block_ids` with `tuple[list[]]` for backward compatibility 

(please polish the statement)